app/gui/src/controller/graph/executed.rs

```diff
@@ -365,6 +365,12 @@ impl Handle {
         self.execution_ctx.set_execution_environment(execution_environment).await?;
         Ok(())
     }
+
+    /// Trigger the execution of the current graph.
+    pub async fn start_execution(&self) -> FallibleResult {
+        self.execution_ctx.start_execution().await?;
+        Ok(())
+    }
```

Comment:
I think we should add somewhere more information about this method. It's not just starting execution (as the execution is actually performed automatically on each change). According to the design. It ought to clear caches and re-execute the entire graph.

Reply:
Is this handled by it using the `live` execution context? If not, we do need some more special handling here. I’ll have a look.

Reply:
I clarified the naming and comments. The clean re-execution should be handled by the API call requesting `language_server::InvalidatedExpressions::All`.

Reply:
I'd call it something like `recompute_in_live_environment` to align with the underlying LS call. AFAIK this is not handled by the `live` environment specifically but rather by the usage of `recompute` with specific arguments (`InvalidatedExpressions::All`)

Reply:
I chose `trigger_clean_live_execution` now, but we can do `recompute_in_live_environment` if that is preferable. I don't have a string opinion either way.